jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/resources/org/jbpm/workbench/ht/client/resources/i18n/Constants.properties

```diff
@@ -134,4 +134,5 @@ Released=Released
 Resumed=Resumed
 Delegated=Delegated
 Nominated=Nominated
+Container_Id= Container Id
```

Comment:
remove extra space

Reply:
ok

Reply:
Regarding the process instance shows this info as 'Deployment' to be consistence with the process instance screen I  would rename this filter too.